models/custom-view/src/custom-view/generator.ts

```diff
@@ -0,0 +1,44 @@
+import { LocalizedField } from '@commercetools-test-data/commons';
+import { fake, Generator } from '@commercetools-test-data/core';
+import { createRelatedDates } from '@commercetools-test-data/utils';
+import { CustomViewPermission } from '../custom-view-permission';
+import { CustomViewTypeSettings } from '../custom-view-type-settings';
+import { CustomView } from './types';
+import { defaultLocators } from './utils';
+
+const [getOlderDate, getNewerDate] = createRelatedDates();
+
+const generator = Generator<CustomView>({
+  fields: {
+    createdAt: fake(getOlderDate),
+    defaultLabel: fake((f) => f.commerce.department()),
+    id: fake((f) => f.string.uuid()),
+    installedBy: null, // buildField
+    labelAllLocales: [
+      LocalizedField.random().build(),
+      LocalizedField.random().build(),
+    ],
```

Comment:
I'm not sure how should I implement these fields. I know we have the `buildFields` available in the transformers file, but I can't make it work.

Some help would be much appreciated.